service/src/main/java/uk/gov/hmcts/reform/fpl/model/OldChild.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OldChild {
+
+    private final String childName;
+    private final Date childDOB;
+    private final String childGender;
+    private final String childGenderIdentification;
+    private final String livingSituation;
+    private final String situationDetails;
+    private final String situationDate;
+    private final String keyDates;
+    private final String careAndContact;
+    private final String adoption;
+    private final String placementOrderApplication;
+    private final String placementCourt;
+    private final String mothersName;
+    private final String fathersName;
+    private final String fathersResponsibility;
+    private final String socialWorkerName;
+    private final String socialWorkerTel;
+    private final String additionalNeeds;
+    private final String additionalNeedsDetails;
+    private final String detailsHidden;
+    private final String detailsHiddenReason;
+    private final String litigationIssues;
+    private final String litigationIssuesDetails;
+    private final Address address;
+
+    @JsonCreator
```

Comment:
I'd normally say remove annotations that are not needed but here I make an exception as that class is subject to removal soon.